packages/frontend-2/components/settings/Dialog.vue

```diff
@@ -56,33 +56,20 @@
               collapsible
             >
               <LayoutSidebarMenuGroupItem
-                v-for="(workspaceMenuItem, itemKey) in menuItemConfig.workspace"
+                v-for="(workspaceMenuItem, itemKey) in workspaceMenuItems"
                 :key="`${key}-${itemKey}`"
                 :label="workspaceMenuItem.title"
-                :class="{
-                  'bg-highlight-2 hover:!bg-highlight-2':
-                    targetMenuItem === itemKey && targetWorkspaceId === workspaceItem.id
-                }"
+                :class="[
+                  targetMenuItem === itemKey &&
```

Comment:
So complicated! Extract to a computed and preferably avoid the return by `&& value`

Reply:
@Mikehrn The underlying problem is that this is too complicated to just be put in the :class attribute as is. If it can't be a computed, it can be a function.

Reply:
Get your point now, updated it :)